backend/src/main/java/com/benine/backend/http/SetCameraInUseHandler.java

```diff
@@ -0,0 +1,43 @@
+package com.benine.backend.http;
+
+import com.benine.backend.camera.Camera;
+import org.eclipse.jetty.server.Request;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/**
+ * Created on 21-05-16.
+ */
+public class SetCameraInUseHandler extends CameraRequestHandler {
+
+  @Override
+  public void handle(String s, Request request, HttpServletRequest req, HttpServletResponse res)
+          throws IOException, ServletException {
+    int camID = getCameraId(request);
+    Camera cam = getCameraController().getCameraById(camID);
+
+    String autoOn = request.getParameter("inuse");
+
+    if (isAllowed(cam) && autoOn != null && !cam.isInUse()) {
```

Comment:
If a client wants to set a camera to not inuse, will it get trough this statement? Because the camera is still in use i think not, and then the inuse cannot be set to false